CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to the LaunchDarkly iOS SDK will be documented in this file.
 ### Multiple Environment clients
 Version 4.0.0 does not support multiple environments. If you use version `2.14.0` or later and set `LDConfig`'s `secondaryMobileKeys` you will not be able to migrate to version `4.0.0`. Multiple Environments will be added in a future release to the Swift SDK.
 
+## [4.1.0] - 2019-06-19
+### Change
+- Installs new `deviceModel` into `EnvironmentReporter` and renames old `deviceModel` to `deviceType`
+- use `CwSysCtl` for macos model
```

Comment:
Is this meant to be two separate entries? They seem related, but the `CrSysCtl` one is rather unclear on what it means.